app/assets/stylesheets/dark-mode.css

```diff
@@ -135,3 +135,6 @@ progress-circle__inner {
 .progress-circle__content {
   color: var(--dark-bg-accent) !important;
 }
+.advert__icon {
+    filter: invert(1);
```

Comment:
Nit: the indentation looks off on this line, we use two spaces across the app.

Reply:
Yes, you're right, and now is fixed 👍🏽 .